openreview/venue_request/venue_request.py

```diff
@@ -185,6 +185,81 @@ def setup_review_stage(self):
             }
         ))
 
+    def setup_official_comment_stage(self):
+
+        official_comment_stage_content = {
+            'commentary_start_date': {
+                'description': 'When does official commentary begin? Please use the following format: YYYY/MM/DD HH:MM (e.g. 2019/01/31 23:59)',
+                'value-regex': r'^[0-9]{4}\/([1-9]|0[1-9]|1[0-2])\/([1-9]|0[1-9]|[1-2][0-9]|3[0-1])(\s+)?((2[0-3]|[01][0-9]|[0-9]):[0-5][0-9])?(\s+)?$',
+                'order': 27
+            },
+            'allow_unsubmitted_reviewers_to_comment': {
+                'description': 'Should the reviewers who have not submitted their review be able to make official comments? Default is "No, unsubmitted reviewers should not be able to post official comments"',
+                'value-radio': [
+                    'Yes, unsubmitted reviewers should be able to post official comments',
+                    'No, unsubmitted reviewers should not be able to post official comments'
+                ],
+                'required': True,
+                'default': 'No, unsubmitted reviewers should not be able to post official comments',
+                'order': 28
+            },
+            'allow_reader_selection': {
+                'description': 'Should the comment author be allowed to select comment readers from a list? Default is "No, comment authors should not be able to select comment readers"',
+                'value-radio': [
+                    'Yes, comment authors should be able to select comment readers',
+                    'No, comment authors should not be able to select comment readers'
+                ],
+                'required': True,
+                'default': 'No, comment authors should not be able to select comment readers',
+                'order': 29
+            },
+            'email_program_chairs_about_official_reviews': {
```

Comment:
what does it mean? official_reviews or official_comments?

Reply:
You are right, it's official comments, sorry.